### PR TITLE
feat: better error message

### PR DESCRIPTION
**Pull request by yqrashawn**
1. better error message
2. use test pk from js sdk

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/470)

### CODE REVIEW

File: packages/rpcs/cfx_sendTransaction/index.js [hunk 1]
```diff
@@ -67,7 +67,16 @@ export const main = async ({
       ])
     } catch (err) {
       if (err?.code === ERROR.USER_REJECTED.code) throw err
-      throw InvalidParams(`Invalid transaction ${JSON.stringify(params)}`)
+      const errmsgPrefix = err?.message?.match(/reverted/i)
```
Comment (zctocm) [inline packages/rpcs/cfx_sendTransaction/index.js:70]: 'reverted' in the place means all the errors about 'reverted'? 
http://developer.confluxnetwork.org/sending-tx/en/transaction_explain/#how-to-check-transaction-details-and-status

**Review by zctocm — APPROVED** [review]

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-2261 使用两个ledger账户发送交易，如果两个账户nonce相同，可以相互发送彼此账户里的token](https://app.clickup.com/t/3719212/PORTAL-2261)

File: packages/rpc-engine/middlewares/validate-rpc-params.js [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
-            '\t',
-          )}\n\nError:\n${JSON.stringify(
-            explain(schemas.input, params, {netId: req.network.netId}),
-            null,
-            '\t',
-          )}`,
-          req,
-        )
+  fn: comp(
+    sideEffect(() => addBreadcrumb({category: 'validateRpcParams'})),
```
Comment (zctocm) [inline packages/rpc-engine/middlewares/validate-rpc-params.js:11]: I am worried about it is not safe that the RPC data will send to Sentry, especially the signature data

Comment (yqrashawn) [reply packages/rpc-engine/middlewares/validate-rpc-params.js:11]: The breadcrumb here is only here to label the to be sent error. 

It's like the trace of the error.

Only data specified in description of breadcrumbs will be sent.

Comment (zctocm) [reply packages/rpc-engine/middlewares/validate-rpc-params.js:11]: ok, maybe similar to that we will know the action path when debugging the error

**Review by zctocm — APPROVED** [review]

**Pull request merged**